src/cluster/Backend.h

```diff
@@ -269,6 +270,16 @@ class Backend {
         return DoPublishLogWrites(header, records);
     }
 
+    /**
+     * @return This backend's implementation name.
+     */
+    const std::string& GetName() const { return name; }
+
+    /**
+     * @return This backend's implementation component tag.
+     */
+    const zeek::Tag& GetTag() const { return tag; }
```

Comment:
Most of the other functions like this are just named `Name()` and `Tag()`. See for example, `Expr::Tag()`, `Stmt::Tag()` and `Type::Tag()`.

Reply:
I've previously added `GetName()`s elsewhere to return `std::string` and deprecated `const char* Name()`s and also got inspired by `Analyzer::GetAnalyzerName()` and `GetAnalyzerTag()` :-)

I'll go for the short ones. Thanks!